djstripe/mixins.py

```diff
@@ -55,5 +55,7 @@ def get_context_data(self, *args, **kwargs):
         context['is_plans_plural'] = bool(len(djstripe_settings.PLAN_CHOICES) > 1)
         context['customer'], created = Customer.get_or_create(
             subscriber=djstripe_settings.subscriber_request_callback(self.request))
-        context['CurrentSubscription'] = CurrentSubscription
+        # Add context variables for both the old CurrentSubscription model and the new Subscription model.
+        context['CurrentSubscription'] = Subscription
```

Comment:
Again, my preference, but I think we should remove backwards compatibility for this.